lib/cryptoexchange/exchanges/coinut/services/market.rb

```diff
@@ -0,0 +1,48 @@
+module Cryptoexchange::Exchanges
+  module Coinut
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          authentication = Cryptoexchange::Exchanges::Coinut::Authentication.new(
+            :market,
+            Cryptoexchange::Exchanges::Coinut::Market::NAME
+          )
+          authentication.validate_credentials!
+
+          payload_ = payload(market_pair)
+          headers = authentication.headers(payload_)
+          output = fetch_using_post(ticker_url, payload_, headers)
+          adapt(output, market_pair)
+        end
+
+        def payload(market_pair)
+          '{"nonce":' + SecureRandom.random_number(99999).to_s + ',"request":"inst_tick", "inst_id":' + market_pair.inst_id + ' }'
```

Comment:
since the format of the payload is similar across Ticker, Order and Trades. Should we move this up into the Authentication class and take in the needed parameters?

Reply:
I would say they are not similar enough